src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -294,6 +290,36 @@ describe('StripeV3PaymentStrategy', () => {
             return expect(response).rejects.toThrow(StandardError);
         });
 
+        it('throws an error when PaymentIntent is undefined', async () => {
```

Comment:
Is this accurate?

Reply:
I still think this needs to be edited before we can merge it.

Reply:
We are not sure if you referring to either verbiage or the actual purpose of the test case @PascalZajac

Reply:
The verbiage - it still refers to PaymentIntent, but we've deleted that afaik, so I believe it should be `ClientToken`?

Reply:
@PascalZajac verbiage updated.